index.html

```diff
@@ -11,12 +11,12 @@
         <link href="https://fonts.googleapis.com/css?family=Roboto%7CRoboto+Condensed" rel="stylesheet">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/katex.min.css" integrity="sha384-dbVIfZGuN1Yq7/1Ocstc1lUEm+AT+/rCkibIcC/OmWo5f0EA48Vf8CytHzGrSwbQ" crossorigin="anonymous">
         <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
-        <link rel="icon" type="image/png" href="/dist/images/favicon-16x16.png">
+        <link rel="icon" type="image/png" href="/src/images/favicon-16x16.png">
```

Comment:
I think this needs to stay as `/dist/`; this is where Webpack copies the files when it compiles

Reply:
I see why this wasn't working for you locally.  The files are referenced outside at this point, so they never get copied to `/src/images/`.  I added a line to the Makefile to fix this.